datadog_checks_dev/datadog_checks/dev/tooling/constants.py

```diff
@@ -33,6 +33,14 @@
     'ntp',
 }
 
+BETA_CHECKS = {
+    'datadog_checks_dev',
```

Comment:
Why is this beta?

Reply:
It's changing a lot. 1.0.0 will happen after e2e is finished